src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -388,34 +388,23 @@ private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(Liter
                             if (validator.test(itemInHand)) {
                                 // Gather a list of the items in the shulker box
                                 ListTag shulkerContents = itemInHand.getTagElement("BlockEntityTag").getList("Items",10);
```

Comment:
This code is not properly formatted

Reply:
I added the space while extracting the code into a method. Please let me know if I'm missing something else here.